packages/devextreme/js/ui/tabs.js

```diff
@@ -529,20 +532,32 @@ const Tabs = CollectionWidget.inherit({
         const navButton = this._createComponent($('<div>').addClass(TABS_NAV_BUTTON_CLASS), Button, {
             focusStateEnabled: false,
             icon: icon,
-            onClick: function() {
-                that._updateScrollPosition(offset, 1);
+            integrationOptions: {},
+            elementAttr: {
+                role: null,
+                'aria-label': null,
+                'aria-disabled': null,
+            },
+            onClick: () => {
+                this._updateScrollPosition(offset, 1);
+            },
+            onOptionChanged: ({ name, value, element }) => {
+                if(name === 'disabled' && value) {
+                    this.setAria({ disabled: null }, $(element));
```

Comment:
According to performance reason it's better to do it in a place where we call option change. 
I understand your idea, this solution keeps a kind of encapsulation and well work if we add one more "disabled" option update. But I think that possible performance issues are more real that this case.

Reply:
[2645c1e4a65e528f6ea25a986348082a6b25187f]